### PR TITLE
Fix documentation in grpc configs

### PR DESCRIPTION
**Pull request by bogdandrutu**

### CODE REVIEW

File: config/configgrpc/configgrpc.go [hunk 1]
```diff
@@ -76,11 +76,11 @@ type GRPCClientSettings struct {
 	// (https://godoc.org/google.golang.org/grpc#WithKeepaliveParams).
 	Keepalive *KeepaliveClientConfig `mapstructure:"keepalive"`
 
-	// The WriteBufferSize for client gRPC. See grpc.WithReadBufferSize
+	// ReadBufferSize for gRPC server. See grpc.WithReadBufferSize
```
Comment (dmitryax) [inline config/configgrpc/configgrpc.go:79]: Should be `client` instead as it's `GRPCClientSettings` struct

File: config/configgrpc/configgrpc.go [hunk 2]
```diff
@@ -147,11 +147,11 @@ type GRPCServerSettings struct {
 	// It has effect only for streaming RPCs.
 	MaxConcurrentStreams uint32 `mapstructure:"max_concurrent_streams"`
 
-	// The WriteBufferSize for client gRPC. See grpc.ReadBufferSize
+	// ReadBufferSize for gRPC client. See grpc.ReadBufferSize
```
Comment (dmitryax) [inline config/configgrpc/configgrpc.go:150]: And `server` here?

File: config/configgrpc/configgrpc.go (hunk 1 above)
Comment (bogdandrutu) [reply config/configgrpc/configgrpc.go]: Done.

File: config/configgrpc/configgrpc.go (hunk 2 above)
Comment (bogdandrutu) [reply config/configgrpc/configgrpc.go]: Done.

**Pull request merged**